cli/tests/command_executor/D.test

```diff
@@ -69,6 +71,7 @@ INSERT INTO foo3 values(5, 1, 1.0, 1.0, 'XYZZ');
  col4   | Float  
  col5   | Char(5)
 ==
+
```

Comment:
In the test, is it possible to also check special chars `\t` and `\r`?

Reply:
Probably no. The test input does not go through `LineReader`.